biome.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.6.1/schema.json",
```

package.json のバージョンとズレていますし，毎回ここも更新するのは大変なので，ローカルで解決するようにしておくことでその手間を省けます．

```suggestion
  "$schema": "node_modules/@biomejs/biome/schema.json",
```